zetaclient/chains/bitcoin/bitcoin.go

```diff
@@ -94,9 +86,9 @@ func (b *Bitcoin) Start(ctx context.Context) error {
 	// Observers
 	register(b.observer.ObserveInbound, "observe_inbound", optInboundInterval, optInboundSkipper)
 	register(b.observer.ProcessInboundTrackers, "process_inbound_trackers", optInboundInterval, optInboundSkipper)
-	register(b.observer.FetchUTXOs, "fetch_utxos", optUTXOInterval, optGenericSkipper)
-	register(b.observer.ObserveMempool, "observe_mempool", optMempoolInterval, optGenericSkipper)
-	register(b.observer.PostGasPrice, "post_gas_price", optGasInterval, optGenericSkipper)
+	register(b.observer.FetchUTXOs, "fetch_utxos", optUTXOInterval, optOutboundSkipper)
+	register(b.observer.ObserveMempool, "observe_mempool", optMempoolInterval, optOutboundSkipper)
```

Comment:
NIT: consider clearly separating out btc and zetacore mempool in variable names to improve code readibility

Reply:
yeah maybe `ObserveBTCMempool` ,

Reply:
https://github.com/zeta-chain/node/pull/4291/commits/ebd44dacf704bed08fee7d50126545c3e8864b24